endpoints/cinema/index.js

```diff
@@ -153,6 +155,7 @@ app.get('/cinema/theaters', (req, res) => {
       })
     })
 
-    return res.cache().json(obj)
+    // return res.cache().json(obj)
+    return res.json(obj)
```

Comment:
We should cache the responses. Please re-enable.

Reply:
Yes, just forgot. Thank you